### ISSUE TITLE
use of quap()

### ISSUE THREAD

**Issue by hannesbecher** [StatisticalRethinkingJulia/StatisticalRethinkingTuring.jl#17]
Hi again,

I can see that `quap()` is defined in StatisticalRethinking.jl but it does not seem to be used in these notebooks here (e.g. Chapter 2, Code chunk 2.6). Is that intentional?

Would it be useful if I tried update the relevant notebooks to using `quap()`? Are there any complications to be expected? No promise but I could give it a go...

All the best,
Hannes

**Comment by goedman** [comment]
Hi Hannes, I’m not near my computer right now but will take a look in a few hours.It would be great to use Turing’s MAP() and/or MLE() functions for those examples. I think at some point I included an example of how to use these. Will get back to you.Thanks again, best,RobSent from my iPhoneOn May 2, 2023, at 09:25, Hannes Becher ***@***.***> wrote:﻿
Hi again,
I can see that quap() is defined in StatisticalRethinking.jl but it does not seem to be used in these notebooks here (e.g. Chapter 2, Code chunk 2.6). Is that intentional?
Would it be useful if I tried update the relevant notebooks to using quap()? Are there any complications to be expected? No promise but I could give it a go...
All the best,
Hannes
